nautobot/extras/tests/test_forms.py

```diff
@@ -131,6 +136,68 @@ def test_create_job_hooks_with_same_job_common_action_same_content_type(self):
         )
 
 
+class NoteModelFormTestCase(TestCase):
+    """
+    TestNoteModelForm validation and saving.
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        active = Status.objects.get(slug="active")
+        cls.user = User.objects.create(username="formmuser1")
+
+        cls.site_form_base_data = {
+            "name": "Site 1",
+            "slug": "site-1",
+            "status": active.pk,
+        }
+
+    def test_note_object_edit_form(self):
+
+        form = SiteForm(data=dict(**self.site_form_base_data, **{"object_note": "This is a test."}))
+        self.assertTrue(form.is_valid())
+        obj = form.save()
+        form.save_note(
+            instance=obj,
+            user=self.user,
+        )
+        self.assertEqual(1, Note.objects.count())
+        self.assertEqual("This is a test.", Note.objects.first().note)
```

Comment:
Perhaps assert that this Note is linked to the correct object instance and user as well?

Reply:
Sure, I will add that test in.